cli/genkey/genkey_test.go

```diff
@@ -1 +1,45 @@
 package genkey
+
+import (
+	"encoding/json"
+	"github.com/cloudflare/cfssl/cli"
+	"os"
+	"os/exec"
+	"path"
+	"testing"
+)
+
+func TestGenkey(t *testing.T) {
+	//testing through console
+	gopath := os.Getenv("GOPATH")
+	cfssl := path.Join(gopath, "bin", "cfssl")
+	testdata := path.Join(gopath, "src", "github.com", "cloudflare", "cfssl", "testdata")
+
+	out1, err := exec.Command(cfssl, "genkey", path.Join(testdata, "csr.json")).Output()
+	var response map[string]interface{}
+	json.Unmarshal(out1, &response)
+
+	if err != nil {
```

Comment:
this should move to be immediately follow where err is assigned, namely at line 19